.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,101 @@ assignees: ''
 
 # Onboarding
 
-By creating this ticket, you are requesting to be onboarded as a developer on the Azul team.
-The action items in this ticket will be carried out by you and existing team
-members.
+This ticket should be created by the hiring supervisor to initiate the process
+of onboarding an employee to team Boardwalk. The action items in this ticket
+will be completed by the supervisor, other team members, and the onboarded
+employee. Your next step is to assign this ticket to yourself.
 
-Your next steps are to carefully read the following documents:
+If you are assigned this ticket, perform the checklist items in order. Remove
+your assignment only after assigning this ticket to the next in line or the
+ticket checklist is completed. At that time, put the ticket back in triage.
 
-- [ ] [Azul README.md](..)
 
-- [ ] [Contributing Guidelines](../blob/develop/CONTRIBUTING.rst)
+## Hiring Supervisor 
 
-- [ ] [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+- [ ] Onboarded employee's name: [ENTER-NAME-HERE]
+- [ ] Approved the HR request for ITS to create a UCSC account for the onboarded employee
+- [ ] Instructed the onboarded employee to create a new GitHub account using the username `{ucsc_account_name}-ucsc` connected to their UCSC account email address. The new account will be used exclusively for work on the team and should be created even if they already have another account.
+- [ ] Assigned this ticket to the onboarded employee
 
-Install the following prerequisites:
+## Onboarded employee actions
 
-- [ ] [git-secrets](..#211-git-secrets)
+- [ ] Your UCSC account email: [ENTER-EMAIL-HERE]
+- [ ] Your GitHub account username: [ENTER-USERNAME-HERE]
+- [ ] Read [Azul README.md](..)
+- [ ] Read [Contributing Guidelines](../blob/develop/CONTRIBUTING.rst)
+- [ ] Read [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+- [ ] Completed the NIH [Information Security and Management Refresher](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Security Awareness for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Management for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the [dbGaP Code of Conduct](https://www.surveymonkey.com/r/HKXNYD7) quiz
+- [ ] Completed the [NIH Security Best Practices for (CAD) Subject to NIH (GDS) Policy](https://www.surveymonkey.com/r/FG3C63T) quiz
+- [ ] Digitally signed the [NIH GDS Policy](https://giwiki.gi.ucsc.edu/images/1/1b/NIH_GDS_Policy.pdf) document and gave it to your project manager
```

Comment:
Confirm with Trevor that this list of training makes a person eligible to access Anvil snapshots.

Reply:
https://ucsc-gi.slack.com/archives/C02B4S0V6GH/p1680108693924329

Ben Vizzier:
> We talked about this a bit in the Boardwalk Standup today.  I suggest that Daniel start with what he has with the understanding that we are reviewing it, and it is likely to change moderately soon.

Nneka:
> i intend to circle back on this as i work on Data Access policy and ToS but will be after both audit